dbt/include/databricks/macros/relations/liquid_clustering.sql

```diff
@@ -0,0 +1,37 @@
+{% macro liquid_clustered_cols() -%}
+  {%- set cols = config.get('liquid_clustered_by', validator=validation.any[list, basestring]) -%}
+  {%- if cols is not none %}
+    {%- if cols is string -%}
+      {%- set cols = [cols] -%}
+    {%- endif -%}
+    CLUSTER BY (
+    {%- for item in cols -%}
+      {{ item }}
+      {%- if not loop.last -%},{%- endif -%}
+    {%- endfor -%}
+    )
+  {%- endif %}
+{%- endmacro -%}
+
+{% macro apply_liquid_clustered_cols(target_relation) -%}
+  {%- set file_format = config.get('file_format', default='delta') -%}
+  {%- set partition_by = config.get('partition_by', validator=validation.any[list, basestring], default=None) -%}
+  {%- set cols = config.get('liquid_clustered_by', validator=validation.any[list, basestring]) -%}
+  {%- if cols is not none %}
+    {%- if cols is string -%}
+      {%- set cols = [cols] -%}
+    {%- endif -%}      
+    {%- call statement('set_cluster_by_columns') -%}
+        ALTER {{ target_relation.type }} {{ target_relation }} CLUSTER BY (
+        {%- for item in cols -%}
+            {{ item }}
+            {%- if not loop.last -%},{%- endif -%}
+        {%- endfor -%}
+        )
+    {%- endcall -%}
+  {%- elif not target_relation.is_hive_metastore() and file_format == "delta" and not partition_by -%}
```

Comment:
If it is possible that this table was previously liquid clustered, and now the user is not specifying cluster by columns, unset the cluster by columns.  Since liquid clustering only works on delta, UC, and without partitions, we only want to run this statement if none of those things are true, otherwise we'll get an error.

Reply:
I saw default of `partition_by` is `None`. shall it be `partition_by is None`?

Reply:
Tried it; unfortunately this is a place where jinja differs from python just a bit, and complains that None is not a test that it knows of.